its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java

```diff
@@ -37,7 +37,7 @@
 class RulingHelper {
 
   private static final String SQ_VERSION_PROPERTY = "sonar.runtimeVersion";
-  private static final String DEFAULT_SQ_VERSION = "LATEST_RELEASE";
+  private static final String DEFAULT_SQ_VERSION = "DEV";
```

Comment:
So now "DEV" is the latest release? Or is it the latest version on master?

Reply:
Actually, you make me notice that I didn't come back to this.
Prior to Orchestrator 5.1, the fix was to use a tag such as `DEV[10.8]`, and if I understand correctly, `LATEST_RELEASE` didn't exist anymore.
With 5.1, `DEV` has now been brought back (latest master), and so was `LATEST_RELEASE`. I shortened `DEV[10.8]` to `DEV` in the PR, but I think what I really wanted to do is indeed to add `LATEST_RELEASE` back...